<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0052)http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3900249/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Warfarin-Amiodarone Drug-Drug Interactions: Determination of [I]u/KI,u for Amiodarone and its Plasma Metabolites</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="./PMC3900249_files/jig.min.css"><script type="text/javascript" src="./PMC3900249_files/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Clinical pharmacology and therapeutics"><meta name="citation_title" content="Warfarin-Amiodarone Drug-Drug Interactions: Determination of [I]u/KI,u for Amiodarone and its Plasma Metabolites"><meta name="citation_authors" content="MG McDonald, NT Au, AK Wittkowsky, AE Rettie"><meta name="citation_date" content="April 2012"><meta name="citation_issue" content="4"><meta name="citation_volume" content="91"><meta name="citation_firstpage" content="709"><meta name="citation_doi" content="10.1038/clpt.2011.283"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3900249/?report=abstract"><meta name="citation_pmid" content="22398967"><meta name="DC.Title" content="Warfarin-Amiodarone Drug-Drug Interactions: Determination of [I]u/KI,u for Amiodarone and its Plasma Metabolites"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="MG McDonald"><meta name="DC.Contributor" content="NT Au"><meta name="DC.Contributor" content="AK Wittkowsky"><meta name="DC.Contributor" content="AE Rettie"><meta name="DC.Date" content="2012 Apr"><meta name="DC.Identifier" content="10.1038/clpt.2011.283"><meta name="DC.Language" content="en"><meta property="og:title" content="Warfarin-Amiodarone Drug-Drug Interactions: Determination of [I]u/KI,u for Amiodarone and its Plasma Metabolites"><meta property="og:type" content="article"><meta property="og:description" content="A retrospective clinical evaluation in a cohort of 73 patients receiving stable anticoagulation therapy showed that the addition/elimination of amiodarone resulted in a 6–65% change in warfarin dose requirement. To evaluate the roles of amiodarone ..."><meta property="og:url" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="http://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="./PMC3900249_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./PMC3900249_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="./PMC3900249_files/common.min.js">//</script><script type="text/javascript" src="./PMC3900249_files/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:&#39;/pmc/utils/ctxp/&#39;"><script type="text/javascript" src="./PMC3900249_files/jquery.citationexporter.js">//</script><link rel="stylesheet" href="./PMC3900249_files/citationexporter.css" type="text/css"><script type="text/javascript" src="./PMC3900249_files/MathJax.js"></script><script type="text/javascript" src="./PMC3900249_files/large-obj-scrollbars.min.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><link rel="alternate" type="application/epub+zip" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/epub/"><link rel="alternate" type="application/pdf" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/pdf/nihms523765.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="http://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="F4FB315376C2946100000000007E87F3"><link type="text/css" rel="stylesheet" href="./PMC3900249_files/3855312.css"><link type="text/css" rel="stylesheet" href="./PMC3900249_files/3929378.css" media="print"><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('http://www.ncbi.nlm.nih.gov/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC3900249%2F&ncbi_pdid=article&ncbi_phid=F4FB315376C2946100000000007E87F3'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
</style></head>
    <body class="article" id="ui-jig-5"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="http://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./PMC3900249_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="http://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a><ul id="resources-submenu" class="ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/all/" tabindex="-1" role="menuitem">All Resources</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#chemicals-bioassays" tabindex="-1" role="menuitem" class="expandRight">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. " tabindex="-1" role="menuitem">BioSystems</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pcassay" title=" Contributed bioactivity data for PubChem Substances " tabindex="-1" role="menuitem">PubChem BioAssay</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pccompound" title=" Unique, standardized chemical structures " tabindex="-1" role="menuitem">PubChem Compound</a></li><li role="presentation"><a href="https://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure " tabindex="-1" role="menuitem">PubChem Structure Search</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pcsubstance" title=" Provides information on the biological activities of small molecules. " tabindex="-1" role="menuitem">PubChem Substance</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/chemicals-bioassays/" tabindex="-1" role="menuitem">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#dna-rna" tabindex="-1" role="menuitem" class="expandRight">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity " tabindex="-1" role="menuitem">BLAST (Basic Local Alignment Search Tool)</a></li><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer " tabindex="-1" role="menuitem">BLAST (Stand-alone)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK25501/" title=" URL application programming interface for searching, linking and downloading data within NCBI&#39;s Entrez system " tabindex="-1" role="menuitem">E-Utilities</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genbank/" title=" DNA and RNA sequences submitted to NCBI " tabindex="-1" role="menuitem">GenBank</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences " tabindex="-1" role="menuitem">GenBank: BankIt</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences " tabindex="-1" role="menuitem">GenBank: Sequin</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data " tabindex="-1" role="menuitem">GenBank: tbl2asn</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/gbench/" title=" Stand-alone application for analyzing sequence data " tabindex="-1" role="menuitem">Genome Workbench</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project " tabindex="-1" role="menuitem">Influenza Virus</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq " tabindex="-1" role="menuitem">Nucleotide Database</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/popset" title=" Sets of nucleotide sequences representing a population " tabindex="-1" role="menuitem">PopSet</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. " tabindex="-1" role="menuitem">Primer-BLAST</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA " tabindex="-1" role="menuitem">ProSplign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI " tabindex="-1" role="menuitem">Reference Sequence (RefSeq)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes " tabindex="-1" role="menuitem">RefSeqGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies " tabindex="-1" role="menuitem">Sequence Read Archive (SRA)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA " tabindex="-1" role="menuitem">Splign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. " tabindex="-1" role="menuitem">Trace Archive</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/unigene" title=" Gene-centered clusters of transcripts and ESTs " tabindex="-1" role="menuitem">UniGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/dna-rna/" tabindex="-1" role="menuitem">All DNA &amp; RNA Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#data-software" tabindex="-1" role="menuitem" class="expandRight">Data &amp; Software</a><ul class="actual-resources-submenu data-software ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity " tabindex="-1" role="menuitem">BLAST (Basic Local Alignment Search Tool)</a></li><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer " tabindex="-1" role="menuitem">BLAST (Stand-alone)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments " tabindex="-1" role="menuitem">Cn3D</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence " tabindex="-1" role="menuitem">Conserved Domain Search Service (CD Search)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK25501/" title=" URL application programming interface for searching, linking and downloading data within NCBI&#39;s Entrez system " tabindex="-1" role="menuitem">E-Utilities</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences " tabindex="-1" role="menuitem">GenBank: BankIt</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences " tabindex="-1" role="menuitem">GenBank: Sequin</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data " tabindex="-1" role="menuitem">GenBank: tbl2asn</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome " tabindex="-1" role="menuitem">Genome ProtMap</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/gbench/" title=" Stand-alone application for analyzing sequence data " tabindex="-1" role="menuitem">Genome Workbench</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. " tabindex="-1" role="menuitem">Primer-BLAST</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA " tabindex="-1" role="menuitem">ProSplign</a></li><li role="presentation"><a href="https://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure " tabindex="-1" role="menuitem">PubChem Structure Search</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. " tabindex="-1" role="menuitem">SNP Submission Tool</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA " tabindex="-1" role="menuitem">Splign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments " tabindex="-1" role="menuitem">Vector Alignment Search Tool (VAST)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/data-software/" tabindex="-1" role="menuitem">All Data &amp; Software Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#domains-structures" tabindex="-1" role="menuitem" class="expandRight">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. " tabindex="-1" role="menuitem">BioSystems</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments " tabindex="-1" role="menuitem">Cn3D</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments " tabindex="-1" role="menuitem">Conserved Domain Database (CDD)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence " tabindex="-1" role="menuitem">Conserved Domain Search Service (CD Search)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/structure/" title=" Database of 3D macromolecular structures derived from PDB " tabindex="-1" role="menuitem">Structure (Molecular Modeling Database)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments " tabindex="-1" role="menuitem">Vector Alignment Search Tool (VAST)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/domains-structures/" tabindex="-1" role="menuitem">All Domains &amp; Structures Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#genes-expression" tabindex="-1" role="menuitem" class="expandRight">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. " tabindex="-1" role="menuitem">BioSystems</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/gap" title=" Genotype/Phenotype association studies " tabindex="-1" role="menuitem">Database of Genotypes and Phenotypes (dbGaP)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK25501/" title=" URL application programming interface for searching, linking and downloading data within NCBI&#39;s Entrez system " tabindex="-1" role="menuitem">E-Utilities</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/gene" title=" Database of gene loci and associated transcripts and proteins " tabindex="-1" role="menuitem">Gene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/geo/" title=" Gene Expression Omnibus (GEO) Database " tabindex="-1" role="menuitem">Gene Expression Omnibus (GEO) Database </a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/gds" title=" Curated collections of gene expression data sets " tabindex="-1" role="menuitem">Gene Expression Omnibus (GEO) Datasets</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/geoprofiles/" title=" Database of gene expression Profiles for individual sequences " tabindex="-1" role="menuitem">Gene Expression Omnibus (GEO) Profiles</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/gbench/" title=" Stand-alone application for analyzing sequence data " tabindex="-1" role="menuitem">Genome Workbench</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/homologene" title=" Sets of homologous eukaryotic genes " tabindex="-1" role="menuitem">HomoloGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data " tabindex="-1" role="menuitem">Map Viewer</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/omim" title=" Database of human genes and genetic disorders " tabindex="-1" role="menuitem">Online Mendelian Inheritance in Man (OMIM)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes " tabindex="-1" role="menuitem">RefSeqGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/unigene" title=" Gene-centered clusters of transcripts and ESTs " tabindex="-1" role="menuitem">UniGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/genes-expression/" tabindex="-1" role="menuitem">All Genes &amp; Expression Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#genetics-medicine" tabindex="-1" role="menuitem" class="expandRight">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books" title=" Online biomedical books " tabindex="-1" role="menuitem">Bookshelf</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/gap" title=" Genotype/Phenotype association studies " tabindex="-1" role="menuitem">Database of Genotypes and Phenotypes (dbGaP)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/gtr/" title=" Detailed information about genetic tests and labs " tabindex="-1" role="menuitem">Genetic Testing Registry</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project " tabindex="-1" role="menuitem">Influenza Virus</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data " tabindex="-1" role="menuitem">Map Viewer</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/omim" title=" Database of human genes and genetic disorders " tabindex="-1" role="menuitem">Online Mendelian Inheritance in Man (OMIM)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pubmed" title=" Biomedical literature abstracts " tabindex="-1" role="menuitem">PubMed</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/" title=" Free, full-text articles " tabindex="-1" role="menuitem">PubMed Central (PMC)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pubmed/clinical" title=" Directory of PubMed queries about clinical topics " tabindex="-1" role="menuitem">PubMed Clinical Queries</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes " tabindex="-1" role="menuitem">RefSeqGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/genetics-medicine/" tabindex="-1" role="menuitem">All Genetics &amp; Medicine Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#genomes-maps" tabindex="-1" role="menuitem" class="expandRight">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/dbvar" title=" A database of genomic structural variation " tabindex="-1" role="menuitem">Database of Genomic Structural Variation (dbVar)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data " tabindex="-1" role="menuitem">GenBank: tbl2asn</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sites/genome" title=" Curated genomic DNA sequences " tabindex="-1" role="menuitem">Genome</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/bioproject/" title=" Central status pages for genome sequencing projects " tabindex="-1" role="menuitem">Genome Project</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome " tabindex="-1" role="menuitem">Genome ProtMap</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/gbench/" title=" Stand-alone application for analyzing sequence data " tabindex="-1" role="menuitem">Genome Workbench</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project " tabindex="-1" role="menuitem">Influenza Virus</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data " tabindex="-1" role="menuitem">Map Viewer</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq " tabindex="-1" role="menuitem">Nucleotide Database</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/popset" title=" Sets of nucleotide sequences representing a population " tabindex="-1" role="menuitem">PopSet</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA " tabindex="-1" role="menuitem">ProSplign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies " tabindex="-1" role="menuitem">Sequence Read Archive (SRA)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA " tabindex="-1" role="menuitem">Splign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. " tabindex="-1" role="menuitem">Trace Archive</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/genomes-maps/" tabindex="-1" role="menuitem">All Genomes &amp; Maps Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#homology" tabindex="-1" role="menuitem" class="expandRight">Homology</a><ul class="actual-resources-submenu homology ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity " tabindex="-1" role="menuitem">BLAST (Basic Local Alignment Search Tool)</a></li><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer " tabindex="-1" role="menuitem">BLAST (Stand-alone)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences " tabindex="-1" role="menuitem">BLAST Link (BLink)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments " tabindex="-1" role="menuitem">Conserved Domain Database (CDD)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence " tabindex="-1" role="menuitem">Conserved Domain Search Service (CD Search)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome " tabindex="-1" role="menuitem">Genome ProtMap</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/homologene" title=" Sets of homologous eukaryotic genes " tabindex="-1" role="menuitem">HomoloGene</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles " tabindex="-1" role="menuitem">Protein Clusters</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/homology/" tabindex="-1" role="menuitem">All Homology Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#literature" tabindex="-1" role="menuitem" class="expandRight">Literature</a><ul class="actual-resources-submenu literature ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books" title=" Online biomedical books " tabindex="-1" role="menuitem">Bookshelf</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK25501/" title=" URL application programming interface for searching, linking and downloading data within NCBI&#39;s Entrez system " tabindex="-1" role="menuitem">E-Utilities</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/nlmcatalog/journals" title=" Information on journals cited by NCBI records " tabindex="-1" role="menuitem">Journals in NCBI Databases</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/mesh" title=" Controlled vocabulary for MEDLINE " tabindex="-1" role="menuitem">MeSH Database</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK143764/" title=" Introductory articles about NCBI resources " tabindex="-1" role="menuitem">NCBI Handbook</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK3831/" title=" Online, searchable help documentation " tabindex="-1" role="menuitem">NCBI Help Manual</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK1969/" title=" NCBI&#39;s monthly newsletter " tabindex="-1" role="menuitem">NCBI News</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pubmed" title=" Biomedical literature abstracts " tabindex="-1" role="menuitem">PubMed</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/" title=" Free, full-text articles " tabindex="-1" role="menuitem">PubMed Central (PMC)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pubmed/clinical" title=" Directory of PubMed queries about clinical topics " tabindex="-1" role="menuitem">PubMed Clinical Queries</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pubmedhealth/" title=" Reviews of clinical effectiveness research " tabindex="-1" role="menuitem">PubMed Health</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/literature/" tabindex="-1" role="menuitem">All Literature Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#proteins" tabindex="-1" role="menuitem" class="expandRight">Proteins</a><ul class="actual-resources-submenu proteins ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. " tabindex="-1" role="menuitem">BioSystems</a></li><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity " tabindex="-1" role="menuitem">BLAST (Basic Local Alignment Search Tool)</a></li><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer " tabindex="-1" role="menuitem">BLAST (Stand-alone)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences " tabindex="-1" role="menuitem">BLAST Link (BLink)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments " tabindex="-1" role="menuitem">Conserved Domain Database (CDD)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence " tabindex="-1" role="menuitem">Conserved Domain Search Service (CD Search)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK25501/" title=" URL application programming interface for searching, linking and downloading data within NCBI&#39;s Entrez system " tabindex="-1" role="menuitem">E-Utilities</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA " tabindex="-1" role="menuitem">ProSplign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles " tabindex="-1" role="menuitem">Protein Clusters</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/protein" title=" Protein sequence data " tabindex="-1" role="menuitem">Protein Database</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI " tabindex="-1" role="menuitem">Reference Sequence (RefSeq)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/proteins/" tabindex="-1" role="menuitem">All Proteins Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#sequence-analysis" tabindex="-1" role="menuitem" class="expandRight">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity " tabindex="-1" role="menuitem">BLAST (Basic Local Alignment Search Tool)</a></li><li role="presentation"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer " tabindex="-1" role="menuitem">BLAST (Stand-alone)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences " tabindex="-1" role="menuitem">BLAST Link (BLink)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence " tabindex="-1" role="menuitem">Conserved Domain Search Service (CD Search)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome " tabindex="-1" role="menuitem">Genome ProtMap</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/gbench/" title=" Stand-alone application for analyzing sequence data " tabindex="-1" role="menuitem">Genome Workbench</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project " tabindex="-1" role="menuitem">Influenza Virus</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. " tabindex="-1" role="menuitem">Primer-BLAST</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA " tabindex="-1" role="menuitem">ProSplign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA " tabindex="-1" role="menuitem">Splign</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/sequence-analysis/" tabindex="-1" role="menuitem">All Sequence Analysis Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#taxonomy" tabindex="-1" role="menuitem" class="expandRight">Taxonomy</a><ul class="actual-resources-submenu taxonomy ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/taxonomy" title=" The NCBI taxonomy tree in Entrez " tabindex="-1" role="menuitem">Taxonomy</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node " tabindex="-1" role="menuitem">Taxonomy Browser</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees " tabindex="-1" role="menuitem">Taxonomy Common Tree</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/taxonomy/" tabindex="-1" role="menuitem">All Taxonomy Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#training-tutorials" tabindex="-1" role="menuitem" class="expandRight">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/education/" title=" Central page providing links to tutorials and training materials " tabindex="-1" role="menuitem">NCBI Education Page</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK143764/" title=" Introductory articles about NCBI resources " tabindex="-1" role="menuitem">NCBI Handbook</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK3831/" title=" Online, searchable help documentation " tabindex="-1" role="menuitem">NCBI Help Manual</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/books/NBK1969/" title=" NCBI&#39;s monthly newsletter " tabindex="-1" role="menuitem">NCBI News</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/training-tutorials/" tabindex="-1" role="menuitem">All Training &amp; Tutorials Resources...</a></li></ul></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#variation" tabindex="-1" role="menuitem" class="expandRight">Variation</a><ul class="actual-resources-submenu variation ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/dbvar" title=" A database of genomic structural variation " tabindex="-1" role="menuitem">Database of Genomic Structural Variation (dbVar)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/gap" title=" Genotype/Phenotype association studies " tabindex="-1" role="menuitem">Database of Genotypes and Phenotypes (dbGaP)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/snp" title=" Database of nucleotide sequence polymorphisms " tabindex="-1" role="menuitem">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. " tabindex="-1" role="menuitem">SNP Submission Tool</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/variation/" tabindex="-1" role="menuitem">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="http://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a><ul id="howto-submenu" class="ui-ncbibasicmenu" role="menu"><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/all/#howtos" tabindex="-1" role="menuitem">All How To</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/chemicals-bioassays/#howtos" tabindex="-1" role="menuitem">Chemicals &amp; Bioassays</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/dna-rna/#howtos" tabindex="-1" role="menuitem">DNA &amp; RNA</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/data-software/#howtos" tabindex="-1" role="menuitem">Data &amp; Software</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/domains-structures/#howtos" tabindex="-1" role="menuitem">Domains &amp; Structures</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/genes-expression/#howtos" tabindex="-1" role="menuitem">Genes &amp; Expression</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/genetics-medicine/#howtos" tabindex="-1" role="menuitem">Genetics &amp; Medicine</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/genomes-maps/#howtos" tabindex="-1" role="menuitem">Genomes &amp; Maps</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/homology/#howtos" tabindex="-1" role="menuitem">Homology</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/literature/#howtos" tabindex="-1" role="menuitem">Literature</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/proteins/#howtos" tabindex="-1" role="menuitem">Proteins</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/sequence-analysis/#howtos" tabindex="-1" role="menuitem">Sequence Analysis</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/taxonomy/#howtos" tabindex="-1" role="menuitem">Taxonomy</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/training-tutorials/#howtos" tabindex="-1" role="menuitem">Training &amp; Tutorials</a></li><li role="presentation"><a href="http://www.ncbi.nlm.nih.gov/guide/variation/#howtos" tabindex="-1" role="menuitem">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="http://www.ncbi.nlm.nih.gov/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="http://www.ncbi.nlm.nih.gov/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="http://www.ncbi.nlm.nih.gov/account/?back_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3900249%2F" id="sign_in">Sign in to NCBI</a><a href="http://www.ncbi.nlm.nih.gov/account/signout/?back_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3900249%2F" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="http://www.ncbi.nlm.nih.gov/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="http://www.ncbi.nlm.nih.gov/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="toolkitbookgh">ToolKitBookgh</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="ui-ncbiclearbutton-wrap box-shadow"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:&#39;pmc-search-autocomplete&#39;,disableUrl:&#39;NcbiSearchBarAutoComplCtrl&#39;" autocomplete="off" data-sbconfig="ds:&#39;no&#39;,pjs:&#39;no&#39;,afs:&#39;yes&#39;" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" style="visibility: hidden;"><img src="./PMC3900249_files/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="http://www.ncbi.nlm.nih.gov/pmc/limits/">Limits</a>
                    </li><li>
                        <a href="http://www.ncbi.nlm.nih.gov/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="http://www.ncbi.nlm.nih.gov/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="http://www.ncbi.nlm.nih.gov/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="http://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="http://www.ncbi.nlm.nih.gov/pmc/?term=hhs%20author%20manuscript[filter]&amp;cmd=search&amp;EntrezSystem2.PEntrez.Pmc.Pmc_LimitsTab.LimitsOff=true" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3900249</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="./PMC3900249_files/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="http://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="http://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="http://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="http://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;" id="ui-jig-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div>Clin Pharmacol Ther. Author manuscript; available in PMC 2014 Jan 23.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22398967" target="pmc_ext" ref="reftype=publisher&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal"><span class="cit">Clin Pharmacol Ther. 2012 Apr; 91(4): 709–717. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Mar 7. </span> <span class="doi">doi:&nbsp; <a href="http://dx.doi.org/10.1038%2Fclpt.2011.283" target="pmc_ext" ref="reftype=other&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.1038/clpt.2011.283</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3900249</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS523765</span></div></div></div></div></div><h1 class="content-title">Warfarin-Amiodarone Drug-Drug Interactions: Determination of [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> for Amiodarone and its Plasma Metabolites</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=McDonald%20M%5Bauth%5D">MG McDonald</a>,<sup>1</sup> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Au%20N%5Bauth%5D">NT Au</a>,<sup>1</sup> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Wittkowsky%20A%5Bauth%5D">AK Wittkowsky</a>,<sup>2</sup> and  <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Rettie%20A%5Bauth%5D">AE Rettie</a><sup>1</sup></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="id557520_ai" style="display:none"><div class="fm-affl" lang="en"><sup>1</sup>Department of Medicinal Chemistry, University of Washington, Seattle, WA</div><div class="fm-affl" lang="en"><sup>2</sup>University of Washington School of Pharmacy, Anticoagulation Services, University of Washington Medical Center, Seattle, WA</div><div id="FN1">Corresponding author: Allan E. Rettie, Department of Medicinal Chemistry, University of Washington, Box 357610, Seattle, WA 98195, USA, tele: (206)685-0615, fax: (206)685-3252, <span class="before-email-separator"></span><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.notgnihsaw.u@eitter" class="oemail">ude.notgnihsaw.u@eitter</a></div></div><div class="togglers"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" class="pmctoggle" rid="id557520_ai">Author information <span>►</span></a> <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" class="pmctoggle" rid="id557520_cpl">Copyright and License information <span>►</span></a></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="id557520_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="http://www.ncbi.nlm.nih.gov/pmc/about/copyright.html">Copyright notice</a>  and <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a></div><div class="fm-copyright half_rhythm">Users may view, print, copy, and download text and data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use:<a href="http://www.nature.com/authors/editorial_policies/license.html#terms" ref="reftype=extlink&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.nature.com/authors/editorial_policies/license.html#terms</a></div></div></div><div class="links-box fm-panel whole_rhythm"><div class="small"><div>The publisher's final edited version of this article is available  at <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22398967" target="pmc_ext" ref="reftype=publisher&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal">Clin Pharmacol Ther</a></div><div>See other articles in PMC that <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="__abstractid206947" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-7"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true" id="ui-jig-8">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__abstractid206947title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">A retrospective clinical evaluation in a cohort of 73 patients receiving stable anticoagulation therapy showed that the addition/elimination of amiodarone resulted in a 6–65% change in warfarin dose requirement. To evaluate the roles of amiodarone and its circulating metabolites in this highly variable inhibitory drug interaction, an LC-ESI<sup>+</sup> MS/MS assay was developed for the quantitation of low concentrations of these compounds in human plasma, utilizing newly synthesized deuterated analogs as internal standards. K<sub>I</sub>’s were determined for the inhibition of (S)-warfarin 7-hydroxylation in human liver microsomes, by parent drug and metabolites, and unbound drug fractions (<em>f</em><sub>u</sub>) were measured so that the ratio of unbound plasma concentration to the microsomal K<sub>I</sub> for unbound drug ([I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub>) could be calculated. From these ratios, we predict a minor metabolite, N,N-didesethylamiodarone, to be a major contributor to the drug interaction between warfarin and amiodarone.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">drug-drug interactions, plasma, in vitro, in vivo, pharmacokinetics, warfarin, inhibition, adverse drug reactions, metabolites</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-9"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-10">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Amiodarone (AMIO) was originally introduced as an anti-anginal drug over 40 years ago (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638482">1</a>), but is now used therapeutically as an effective Class III antiarrhythmic agent (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638508">2</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638464">3</a>). AMIO is associated with numerous potential side effects that include pulmonary toxicity (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638474">4</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638512">5</a>), hepatotoxicity (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638476">6</a>) and thyroid dysfunction (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638479">7</a>). Additionally, there have been reports of adverse drug interactions when AMIO is administered together with a wide range of other therapeutic agents including theophylline (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638484">8</a>), flecainide (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638505">9</a>), cyclosporine A (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638480">10</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638503">11</a>) and dextromethorphan (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638472">12</a>). However, the most common, as well as potentially the most dangerous drug interaction that AMIO exhibits is its potentiation of the anticoagulant effect of warfarin, which greatly increases the patient’s risk of hemorrhage (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638490">8</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638481">13</a>).</p><p id="P3">AMIO is frequently used for rhythm control in patients with atrial fibrillation (AF). Stroke is a major concern in patients with AF and since warfarin is the preferred treatment for stroke prevention, AMIO is therefore often co-administered with warfarin (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638502">14</a>). An interaction always occurs between these two drugs (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638475">15</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638466">16</a>) that typically necessitates a dose reduction in warfarin of 25–40% depending on the amiodarone maintenance dose (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638489">15</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638463">16</a>).</p><p id="P4">Early studies by Trager, O’Reilly and co-workers demonstrated that AMIO inhibits the clearance of both (S)- and (R)-warfarin (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638467">17</a>) and decreases the formation clearances of (S)-warfarin metabolites to a greater extent than (R)-warfarin metabolites (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638488">18</a>). However, AMIO proved to be a relatively weak inhibitor of (S)-warfarin metabolism in human liver microsomes (HLM) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638495">18</a>) and AMIO’s N-dealkylated metabolite, monodesethylamiodarone (MDEA), is 40–90 times more potent an inhibitor of CYP2C9 than the parent drug (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638510">19</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638511">20</a>). These observations are significant because CYP2C9 is the major P450 that terminates the pharmacological activity of warfarin (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638483">21</a>). Moreover, MDEA is present in human plasma at a concentration approximately equal to that of the parent drug (~1–5 μM) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638477">4</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638507">22</a>). Indeed, there are in vivo data which suggest that plasma MDEA concentration may be a better predictor of the change in INR than plasma AMIO concentration in patients receiving both drugs (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638492">23</a>). These observations prompt the hypothesis that a major mechanism of the warfarin-AMIO interaction is inhibition of CYP2C9-mediated (S)-warfarin metabolism by MDEA.</p><p id="P5">Although MDEA is the major metabolite of AMIO in humans, a number of others have recently been identified in plasma (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638469">24</a>). The most prominent of these metabolites include 3′-hydroxy-N-monodesethylamiodarone (3′-OHMDEA), N,N-didesethylamiodarone (DDEA) and deaminated amiodarone (DAA). Another potential product of AMIO metabolism is the O-desalkylated derivative (ODAA) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R25" rid="R25" class=" bibr popnode">25</a>). It is possible that one or more of these minor AMIO metabolites could contribute to the warfarin drug interaction and, in fact, several close analogs of ODAA have been shown to be low nanomolar inhibitors of CYP2C9 (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638470">26</a>). Delineation of specific inhibitory AMIO metabolites may also assist in a better understanding of inter-individual variability in the magnitude of this drug-drug interaction (DDI).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><img src="./PMC3900249_files/nihms523765f1.gif" class="small-thumb" alt="Figure 1" title="Figure 1" src-large="/pmc/articles/PMC3900249/bin/nihms523765f1.jpg"></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a4--><div><span>Amiodarone metabolites in human plasma.</span></div></div></div><p id="P6" class="p p-last">The aims of this research were threefold. Firstly, a retrospective study was performed to determine mean and range of warfarin dose change required to maintain an INR of 2.0 – 3.0 after amiodarone was added or discontinued in a cohort of patients undergoing stable warfarin therapy. Next, stabilized plasma concentrations, [I], of AMIO and five of its circulating metabolites were determined in several warfarin patients undergoing combination therapy. In order to measure low metabolite concentrations in vivo, we developed a new, highly sensitive stable-label isotope LC-MS assay that necessitated the synthesis of deuterium-labeled internal standards. Finally, in vitro inhibition experiments were carried out with AMIO and its metabolites to measure the K<sub>I</sub> against CYP2C9-mediated (S)-warfarin 7-hydroxylation in HLM. To best assess the likelihood of an in vivo drug interaction, we measured the free fraction of AMIO and its various metabolites in HLM and in plasma in order to calculate the respective [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> values for parent drug and circulating metabolites.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-11"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-12">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Results</h2><div id="S3" class="sec sec-first"><h3>In vivo Effect of AMIO on Warfarin Dose Requirement</h3><p id="P7" class="p p-first-last">Over the six year period of the retrospective clinical evaluation, 73 case studies were identified fitting the parameters of our analysis for an out of range INR attributed to a warfarin-AMIO drug interaction which occurred upon either addition or elimination of AMIO therapy to patients undergoing stable warfarin treatment. Results of the analysis are described in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The mean change in warfarin dose required to maintain an INR of 2–3 was 25.6%. Warfarin dosing change requirements were highly variable, with a range of 5.9% to 65%.</p><!--table ft1--><!--table-wrap mode=article t1--><div class="table-wrap iconblock ten_col whole_rhythm clearfix" id="T1"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T1/" target="table" rid-ob="ob-T1" rid-figpopup="T1" class="table img_link icnblk_img figpopup"><img alt="Table 1" title="Table 1" class="small-thumb" src="./PMC3900249_files/saved_resource" src-large="/pmc/articles/PMC3900249/table/T1/?report=previmg"></a><div class="icnblk_cntnt"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T1/" target="table" rid-figpopup="T1" rid-ob="ob-T1">Table 1</a></div><!--caption a4--><div><span>Demographic characteristics of patients, and change in warfarin dose required to maintain INR 2.0 – 3.0 after addition or discontinuation of amiodarone.</span></div></div></div></div><div id="S4" class="sec"><h3>Synthesis of AMIO Metabolite Standards</h3><p id="P8" class="p p-first-last">To enable the most rigorous quantitative assessment of AMIO metabolite concentrations in vivo and in vitro, we adopted stable-isotope methodology for all the LC/MS analyses. Literature procedures exist for the synthesis of unlabeled AMIO and many of the drug’s known human plasma metabolites (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R27" rid="R27" class=" bibr popnode">27</a>–<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R30" rid="R30" class=" bibr popnode">30</a>), but the syntheses of deuterium labeled standards for AMIO, MDEA, DDEA, ODAA and DAA are reported here for the first time. The deuterium source for isotope-labeled AMIO (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638487">7</a>), as well as its deaminated (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638473">6</a>) and O-desalkylated (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638471">5</a>) derivatives, was <em>p</em>-methoxy-(2,3,5,6-d<sub>4</sub>)-benzoic acid, <strong>1</strong> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Scheme 1A</span></a>). A slightly different methodology was used in the syntheses of the isotopically-labeled analogs of MDEA and DDEA, <strong>9</strong> and <strong>11</strong>, which were both derived from unlabeled ODAA utilizing 1,2-dibromo-d<sub>4</sub>-ethane as the deuterium source (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Scheme 1B</span></a>). These procedures provided the required stable-isotope internal standards in good yield and purity, and with high deuterium contents (&gt;95%) that reflected those of the labeled starting materials.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><img src="./PMC3900249_files/nihms523765f5.gif" class="small-thumb" alt="Scheme 1" title="Scheme 1" src-large="/pmc/articles/PMC3900249/bin/nihms523765f5.jpg"></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Scheme 1</a></div><!--caption a4--><div><span>Synthesis of deuterium-labeled AMIO and circulating AMIO metabolites</span></div></div></div></div><div id="S5" class="sec"><h3>Measurement of AMIO Parent Drug and Metabolite Concentrations in Human Plasma</h3><p id="P9" class="p p-first-last">The deuterated standards of AMIO and its metabolites were used to develop an LC-ESI<sup>+</sup> MS/MS assay for the quantitation of these compounds in the plasma of clinical patients undergoing concomitant warfarin and AMIO therapy (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Blood was drawn from three patients at various time points up to 14 weeks after initiation of AMIO treatment. Parent drug and metabolite plasma concentrations were seen to plateau at around 8 weeks after addition of AMIO therapy to a stabilized warfarin regime (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>), therefore, average drug and metabolite plasma concentrations were calculated for each of the patients between 8 and 14 weeks, and these values were then averaged for the 3 patients to determine the mean steady state plasma concentrations for AMIO, MDEA, DDEA, 3′-OHMDEA, DAA and ODAA (1.09 ± 0.09, 1.16 ± 0.08, 0.059 ± 0.003, 0.42 ± 0.34, 0.015 ± 0.003 and 0.11 ± 0.01 μM, respectively, after converting units from ng/mL, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The values for AMIO, MDEA, 3′-OHMDEA, DDEA and DAA are similar to those reported in an earlier study by Ha et al., (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638468">24</a>). Plasma ODAA concentrations are reported here for the first time.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><img src="./PMC3900249_files/nihms523765f2.gif" class="small-thumb" alt="Figure 2" title="Figure 2" src-large="/pmc/articles/PMC3900249/bin/nihms523765f2.jpg"></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a4--><div><span>LC-ESI<sup>+</sup> MS/MS trace analysis of a standard mix containing d<sub>2</sub>-ODAA, d<sub>2</sub>-DAA, d<sub>2</sub>-AMIO, d<sub>4</sub>-MDEA, d<sub>4</sub>-DDEA and unlabeled 3′-OHMDEA. No overlap of unlabeled material into the channels measuring deuterated standards could be seen above background within</span> <strong>...</strong></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><img src="./PMC3900249_files/nihms523765f3.gif" class="small-thumb" alt="Figure 3" title="Figure 3" src-large="/pmc/articles/PMC3900249/bin/nihms523765f3.jpg"></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a4--><div><span>Average plasma concentration vs time curves for AMIO and its circulating metabolites in patients (n = 3), already undergoing warfarin treatment, who were started on AMIO therapy. Error bars denote standard error measurements. AMIO dose varied from 230</span> <strong>...</strong></div></div></div><!--table ft1--><!--table-wrap mode=article t1--><div class="table-wrap iconblock ten_col whole_rhythm clearfix" id="T2"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="table" rid-ob="ob-T2" rid-figpopup="T2" class="table img_link icnblk_img figpopup"><img alt="Table 2" title="Table 2" class="small-thumb" src="./PMC3900249_files/saved_resource(1)" src-large="/pmc/articles/PMC3900249/table/T2/?report=previmg"></a><div class="icnblk_cntnt"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="table" rid-figpopup="T2" rid-ob="ob-T2">Table 2</a></div><!--caption a4--><div><span>Comparison of [I]/Ki and [I]<sub>u</sub>/Ki,<sub>u</sub> values for the inhibition of (S)-warfarin 7-hydroxylation by AMIO and its circulating human metabolites.</span></div></div></div></div><div id="S6" class="sec"><h3>Inhibition of (S)-Warfarin 7-Hydroxylation in HLM</h3><p id="P10" class="p p-first-last">Kinetic studies were carried out to determine the enzymatic mechanism of inhibition and to measure the K<sub>I</sub> values for the inhibition of (S)-warfarin metabolism by AMIO and its metabolites, in pooled HLM. Both ODAA (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>) and DAA (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4B</span></a>) were found to be purely competitive inhibitors of (S)-warfarin 7-hydroxylation, with K<sub>I</sub>’s of 0.032 ± 0.006 and 2.0 ± 0.3 μM, respectively, while AMIO, MDEA, DDEA and 3′-OHMDEA all appeared to be mixed inhibitors of warfarin metabolism in HLM (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4C–F</span></a>). The respective K<sub>I</sub> values for these four nitrogen-bearing compounds, determined from Dixon plots, were calculated to be 39 ± 14, 5.9 ± 0.9, 0.053 ± 0.008 and 5.2 ± 2.0 μM, respectively. Due to extensive non-specific protein binding, IC50 values for the test compounds varied significantly, and linearly, with the amount of microsomal protein that was used in the incubations (data not shown). Therefore, the kinetic estimates presented here are K<sub>I</sub> (apparent) values and need to be corrected for the free fractions available in metabolic incubations containing 0.25 mg/mL microsomal protein.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><img src="./PMC3900249_files/nihms523765f4.gif" class="small-thumb" alt="Figure 4" title="Figure 4" src-large="/pmc/articles/PMC3900249/bin/nihms523765f4.jpg"></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a4--><div><span>Lineweaver-Burke (L-B) reciprocal plots of the velocity of formation of (S)-7-hydroxywarfarin vs. (S)-warfarin concentration (1, 4 and 16 μM) in the presence of fixed concentrations of A) ODAA, B) DAA, C) AMIO, D) MDEA, E) DDEA or F) 3′-OHMDEA</span> <strong>...</strong></div></div></div></div><div id="S7" class="sec"><h3>Determination of Fraction Unbound (f<sub>u</sub>) for Parent Drug and Metabolites</h3><p id="P11" class="p p-first">Protein binding experiments demonstrated that AMIO and the majority of its metabolites are very highly bound in both HLM and human plasma. In experiments with HLM, carried out at 0.25 mg/mL microsomal protein, <em>f</em><sub>u</sub> values of 1.4 ± 0.2, 1.2 ± 0.4, 1.1 ± 0.3 and 1.7 ± 0.1% were calculated for AMIO, MDEA, DDEA and DAA, while ODAA and 3′-OHMDEA exhibited <em>f</em><sub>u</sub> values of 5.0 ± 2.0 and 17 ± 2.8%, respectively. The degree to which most of these compounds bind to plasma protein is slightly less than their protein binding to HLM with calculated plasma <em>f</em><sub>u</sub> values of 3.3 ± 0.8, 2.0 ± 0.40, 4.2 ± 0.4 and 4.9 ± 1.7% for AMIO, MDEA, DDEA and DAA, respectively. ODAA and 3′-OHMDEA do not fit this trend as both compounds are much more tightly bound to plasma protein than to microsomal protein, exhibiting plasma <em>f</em><sub>u</sub> values of 0.13 ± 0.01 and 1.8 ± 0.3%, respectively (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P12" class="p p-last">Similar protein binding experiments for these compounds were carried out in the presence of either 4 μM (S)-warfarin, for studies in HLM, or 2 μg/mL rac-warfarin (approximate physiological concentration), for studies in human plasma. Warfarin was found not to affect metabolite protein binding in either HLM or plasma as the fraction unbound values remained essentially unchanged from the numbers reported above, showing less than 2-fold variation in all cases (data not shown).</p></div><div id="S8" class="sec sec-last"><h3>Determination of [I]/K<sub>I</sub> and [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> for Parent Drug and Metabolites</h3><p id="P13" class="p p-first-last">[I]/K<sub>I</sub>, defined here as the total mean plasma drug concentration for the three test subjects after reaching plateau concentrations, i.e. 8–14 weeks after initiation of therapy, divided by the in vitro K<sub>I</sub> (apparent), measured for the inhibition of (S)-warfarin 7-hydroxylation in HLM, was calculated to be 0.028, 0.20, 1.1, 3.4, 0.0076 and 0.081 for AMIO, MDEA, DDEA, ODAA, DAA and 3′-OHMDEA, respectively (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> is defined as the ratio of unbound plasma drug concentration ([I]<sub>u</sub> = [I] * <em>f</em><sub>u</sub><sup>plasma</sup>) to the microsomal K<sub>I</sub> corrected for unbound microsomal drug concentration (K<sub>I</sub>,<sub>u</sub> = K<sub>I</sub> * <em>f</em><sub>u</sub><sup>mics</sup>). These values were determined as 0.066, 0.33, 4.3, 0.089, 0.022 and 0.0086 for AMIO, MDEA, DDEA, ODAA, DAA and 3′-OHMDEA, respectively (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-13"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-14">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Discussion</h2><p id="P14" class="p p-first">The warfarin-AMIO interaction is known to be due to metabolic inhibition of the P450 enzymes responsible for the clearance of warfarin, mainly CYP2C9 (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638491">18</a>). In patient studies, we confirmed (i) the wide inter-individual variability in the magnitude of the warfarin-AMIO interaction reported initially by Kerin et al., (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638465">15</a>), and (ii) the plasma concentration ranges of AMIO and several metabolites reported earlier by Ha et al. (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638486">24</a>). Additionally, in these studies, ODAA was quantified (60.5 ± 5.6 ng/mL, 0.11 ± .01 μM) for the first time in human plasma (although very recently, Deng and coworkers identified ODAA in human bile and as a product of metabolic incubations of AMIO with HLMs (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R31" rid="R31" class=" bibr popnode">31</a>)). The results of the in vitro inhibition and protein binding experiments with AMIO and its metabolites that we report here serve as a means of elucidating details of the molecular mechanism behind this in vivo interaction.</p><p id="P15">AMIO is known to be highly protein bound in human plasma and the <em>f</em><sub>u</sub> measured in plasma for this study (3.3 ± 0.8%) is in excellent agreement with the value reported previously by Lalloz, et al., (3.7 ± 0.6%) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638506">32</a>). Both values were determined by the ultracentrifugation method for measuring free drug fraction. The plasma free fractions for the metabolites of amiodarone - MDEA, 3′-OHMDEA, DDEA, ODAA and DAA, which varied from ~0.1–5% – have not been previously reported.</p><p id="P16">Kinetic experiments carried out in HLM, at 0.25 mg/mL microsomal protein, yielded K<sub>I</sub> (apparent) values of 39, 5.9, 5.2, 0.053, 0.032 and 2.0 μM for the inhibition of (S)-warfarin 7-hydroxylation by AMIO, MDEA, 3′-OHMDEA, DDEA, ODAA and DAA. Both the phenol, ODAA, and the alcohol, DAA, were competitive inhibitors of (S)-warfarin metabolism in HLM while the amine compounds AMIO, MDEA, 3′-OHMDEA and DDEA were mixed inhibitors of (S)-warfarin 7-hydroxylation (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). Ohyama, et al., previously reported AMIO to be a noncompetitive inhibitor and MDEA to be a mixed inhibitor of (S)-warfarin metabolism in microsomes derived from human B-lymphoblastoid cells co-expressing CYP2C9 and P450 reductase, with K<sub>I</sub> values of 95 and 2.3 μM (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638499">20</a>).</p><p id="P17">At present, the U.S. Food and Drug Administration guidance for the use of in vitro data to predict in vivo DDIs relies on the comparison of total inhibitor plasma concentration to in vitro inhibition constant, generally measured in microsomes, ([I]/K<sub>I</sub>) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R33" rid="R33" class=" bibr popnode">33</a>). Compounds which exhibit [I]/K<sub>I</sub> values &lt; 0.1 are considered remote possibilities to cause an interaction, whereas compounds with [I]/K<sub>I</sub> between 0.1 and 1.0 and [I]/K<sub>I</sub> &gt; 1.0 are, respectively, considered possible and likely contributors to potential in vivo interactions (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R33" rid="R33" class=" bibr popnode">33</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638498">34</a>). Using this measure, our data suggest that both DDEA and ODAA are likely to cause a DDI with warfarin, while MDEA is a possible contributor.</p><p id="P18">There are, however, potential problems with using [I]/K<sub>I</sub> for prediction of in vivo interactions, especially for compounds that are highly protein-bound in plasma and microsomes. For these compounds, [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub>, which corrects for the amount of free drug available in plasma and also normalizes the microsomal inhibition constant for the amount of free drug available in microsomes, is accepted as a more accurate predictor of a drug’s potential to cause an in vivo interaction (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638493">35</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638496">36</a>). In comparing the two ratios, [I]/K<sub>I</sub> appears to both under-predict the potential of DDEA to cause an interaction while greatly over-predicting the potential in vivo interaction of ODAA, which changes from likely to remote when [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> is assessed. By either predictive method, MDEA is seen as a possible contributor, while AMIO, DAA and 3′-OHMDEA would be judged as unlikely to contribute to the potentiation of the warfarin anticoagulant effect. Also, though warfarin binds very strongly to plasma (<em>f</em><sub>u(serum)</sub> = 0.008) (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638494">37</a>), the anticoagulant was unable to displace AMIO or its metabolites from plasma protein when incubated at a physiological concentration, showing essentially no effect on <em>f</em><sub>u</sub> for any of the compounds tested.</p><p id="P19">The overall change in intrinsic clearance of (S)-warfarin caused by the drug interaction with AMIO can be predicted according to <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#FD1" rid="FD1" class=" disp-formula">equation 1</a>.</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0"><nobr><span class="math" id="M1" overflow="scroll" role="math" style="width: 27.22em; display: inline-block;"><span style="display: inline-block; position: relative; width: 21.928em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.615em 1021.93em 2.926em -999.997em); top: -2.518em; left: 0.003em;"><span class="mrow" id="MathJax-Span-2"><span class="msub" id="MathJax-Span-3"><span style="display: inline-block; position: relative; width: 2.119em; height: 0px;"><span style="position: absolute; clip: rect(3.178em 1001.26em 4.136em -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-4" style="font-family: STIXGeneral-Regular;">CL</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 1.263em;"><span class="mo" id="MathJax-Span-5" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">int</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-6" style="font-family: STIXGeneral-Regular;">/</span><span class="msub" id="MathJax-Span-7"><span style="display: inline-block; position: relative; width: 4.791em; height: 0px;"><span style="position: absolute; clip: rect(3.178em 1001.26em 4.136em -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-8" style="font-family: STIXGeneral-Regular;">CL</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 1.263em;"><span class="mrow" id="MathJax-Span-9"><span class="mo" id="MathJax-Span-10" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">int</span><span class="mo" id="MathJax-Span-11" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">,</span><span class="mtext" id="MathJax-Span-12" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">inhibited</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-13" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">=</span><span class="msub" id="MathJax-Span-14" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 4.69em; height: 0px;"><span style="position: absolute; clip: rect(3.178em 1002.07em 4.136em -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-15" style="font-family: STIXGeneral-Regular;">AUC</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 2.119em;"><span class="mtext" id="MathJax-Span-16" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">inhibited</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-17" style="font-family: STIXGeneral-Regular;">/</span><span class="mtext" id="MathJax-Span-18" style="font-family: STIXGeneral-Regular;">AUC</span><span class="mo" id="MathJax-Span-19" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">=</span><span class="mn" id="MathJax-Span-20" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">1</span><span class="mo" id="MathJax-Span-21" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="mi" id="MathJax-Span-22" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">∑</span><span class="mspace" id="MathJax-Span-23" style="height: 0.003em; vertical-align: 0.003em; width: 0.204em; display: inline-block; overflow: hidden;"></span><span class="mo" id="MathJax-Span-24" style="font-family: STIXGeneral-Regular;">[</span><span class="mi" id="MathJax-Span-25" style="font-family: STIXGeneral-Regular;">I</span><span class="mo" id="MathJax-Span-26" style="font-family: STIXGeneral-Regular;">]</span><span class="mo" id="MathJax-Span-27" style="font-family: STIXGeneral-Regular;">/</span><span class="msub" id="MathJax-Span-28"><span style="display: inline-block; position: relative; width: 1.011em; height: 0px;"><span style="position: absolute; clip: rect(3.178em 1000.71em 4.136em -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-29" style="font-family: STIXGeneral-Regular;">K<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mi" id="MathJax-Span-30" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">I</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.523em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.372em; border-left-width: 0px; border-left-style: solid; width: 0px; height: 1.378em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><msub><mtext>CL</mtext><mo>int</mo></msub><mo>/</mo><msub><mtext>CL</mtext><mrow><mo>int</mo><mo>,</mo><mtext>inhibited</mtext></mrow></msub><mo>=</mo><msub><mtext>AUC</mtext><mtext>inhibited</mtext></msub><mo>/</mo><mtext>AUC</mtext><mo>=</mo><mn>1</mn><mo>+</mo><mi mathvariant="normal">∑</mi><mspace width="0.16667em"></mspace><mo stretchy="false">[</mo><mi mathvariant="normal">I</mi><mo stretchy="false">]</mo><mo>/</mo><msub><mi mathvariant="normal">K</mi><mi mathvariant="normal">I</mi></msub></math></script></div><div class="l" style="width:8.40em">(Equation 1)</div></div><p id="P20">A 6-fold increase in area under the plasma concentration vs time curve (AUC) is therefore predicted for (S)-warfarin clearance based on the sum of the [I]/K<sub>I</sub> ratios of the parent drug and circulating metabolites (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Serendipitously, if [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> ratios are used instead of [I]/K<sub>I</sub>, the same 6-fold increase in AUC is predicted, however, the prediction using [I]/K<sub>I</sub> is dominated by the ODAA ratio while the prediction based on [I]<sub>u</sub>/K<sub>I</sub>,<sub>u</sub> values is due almost entirely to DDEA (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P21">The actual increase in the mean AUC for (S)-warfarin caused by the drug interaction with AMIO has been reported to range from 27 to 110% in single warfarin dose studies carried out in small groups (n = 5 or 6) of healthy volunteers (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638497">17</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638478">18</a>). The data from our retrospective clinical evaluation showing that introduction/discontinuation of AMIO therapy results in a 6 – 65% change in warfarin dose requirement is in good agreement with this clearance data. However, it is also clear that our in vitro data over-predict the inhibitory effect of the overall drug interaction. Tertiary amines are increasingly recognized as mechanism-based inactivators of P450 following sequential metabolism to C-nitroso metabolites whose nitrogen lone pair binds tightly to the reduced heme (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638501">38</a>). Since AMIO and its aminated metabolites are mixed inhibitors of CYP2C9, it is possible that one or more of these compounds could be a mechanism-based inhibitor (MBI) of the enzyme. In fact, Mori, et al. (2009) found AMIO and MDEA to be (weak) MBIs of CYP2C9-mediated diclofenac 4′-hydroxylation, both exhibiting K<sub>I</sub>’s of inactivation of ~100 μM and k<sub>inact</sub> values of only ~0.1/min (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638504">39</a>). However, given that we conducted our kinetic inhibition experiments with AMIO and MDEA at concentrations that were much lower than the K<sub>I</sub> values reported by Mori et al. (with one exception - for AMIO itself we used a maximum concentration approximately equal to the reported K<sub>I</sub>), it seems unlikely that time-dependent inhibition of microsomal CYP2C9 is a complicating feature of the kinetic plots obtained under the conditions used here with either the parent drug or its primary metabolite. Nonetheless, studies are underway to evaluate the potential of each of the AMIO metabolites described here to evoke time-dependent P450 inhibition. It is possible that the discrepancy in our prediction of the DDI may be explained by high levels of drug/metabolite efflux resulting in decreased hepatocyte exposure. The recent findings of Deng and coworkers, identifying all of the known circulating AMIO metabolites, (as well as a host of new metabolites) in human bile lends support to this hypothesis (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R31" rid="R31" class=" bibr popnode">31</a>)</p><p id="P22" class="p p-last">European Medicines Agency guidelines currently recommend that only metabolites of drugs that circulate at unbound or total molar concentrations of ≥ 20% of the parent drug be tested for the potential to cause DDIs (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R40" rid="R40" class=" bibr popnode">40</a>). Based on our results, DDEA appears to be the most likely metabolite to potentiate the anticoagulant effect when AMIO and warfarin are given in combination, even though AMIO plasma concentrations were roughly 20-fold higher than DDEA concentrations in the patients studied. From our data it would appear that, at least in the specific case of drugs that contain secondary or tertiary amines, the current prediction system might be improved by the inclusion of mono- and especially di-N-desalkyl derivatives – even if these metabolites circulate at very low concentrations – since they are potentially much stronger P450 inhibitors than the parent drug. Studies are underway to determine whether patient variability in formation of DDEA, alone or in combination with other inhibitory metabolites, explains inter-individual variability in the drug interaction between AMIO and warfarin.</p></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-15"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-16">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Methods</h2><div id="S11" class="sec sec-first"><h3>Retrospective Clinical Evaluation of the AMIO/Warfarin Drug-Interaction</h3><p id="P23" class="p p-first-last">From 7/1/2001 to 6/30/2007, a university-affiliated anticoagulation clinic provided routine outpatient anticoagulation management for 3129 patients taking warfarin. During this 6 year period, a total of 230 incidents in which an out-of-range INR was attributed to the warfarin-amiodarone drug interaction were identified. These incidents were associated with 163 episodes in which amiodarone was added or discontinued in patients taking warfarin, or in which the amiodarone dose was changed. Of these 163 episodes, 90 were excluded from analysis due to either unstable warfarin therapy prior to addition of amiodarone (n=40), an interaction due to change in amiodarone dose (n=27); warfarin and amiodarone having been initiated concurrently (n=14), the duration of concurrent warfarin/amiodarone therapy being less than 1 month (n=6) or no records available (n=3). The 73 remaining patients were mainly White (n=66), with Blacks (n = 3) and Asians (n = 4) also represented in the cohort.</p></div><div id="S12" class="sec"><h3>Recruitment of Warfarin Patients for Analysis of Amiodarone Plasma Metabolites</h3><p id="P24" class="p p-first-last">Three patients stabilized on warfarin who were to be initiated on concomitant AMIO therapy were recruited through Cardiology and the Anticoagulation Clinic at the University of Washington Medical Center. Briefly, blood was drawn at regular clinic visits for measurement of INR immediately prior to, and over a 14 week period after, addition of AMIO to the treatment regimen. Introduction of AMIO in the three patients resulted in a 33, 40 or 71% reduction in required warfarin dose. All patients gave informed consent for the study which was approved by the University of Washington IRB.</p></div><div id="S13" class="sec"><h3>AMIO Metabolite Quantitative LCMS Assay</h3><p id="P25" class="p p-first">LCMS analyses were conducted on a Micromass Quattro Premier XE Tandem Quadrupole Mass Spectrometer (Micromass Ltd., Manchester, U.K.) coupled to an ACQUITY Ultra Performance LC<sup>™</sup> (UPLC<sup>™</sup>) System with integral autoinjector (Waters Corp., Milford, MA). The Premier XE was run in ESI<sup>+</sup>-MS/MS MRM mode at a source temperature of 100 °C and a desolvation temperature of 34 °C. The cone voltage was set at 25 Volts while the collision energy was set to 40 Volts. The following mass transitions were monitored in separate ion channels: <em>m/z</em> 547 to 373 (for d<sub>0</sub>-ODAA), 549 to 375 (d<sub>2</sub>-ODAA), 590 to 547 (d<sub>0</sub>-DDEA), 594 to 547 (d<sub>4</sub>-DDEA), 591 to 547 (d<sub>0</sub>-DAA), 593 to 549 (d<sub>2</sub>-DAA), 618 to 547 (d<sub>0</sub>-MDEA), 622 to 547 (d<sub>4</sub>-MDEA), 646 to 547 (d<sub>0</sub>-AMIO), 648 to 549 (d<sub>2</sub>-AMIO) and 634 to 545 (3′-OHMDEA). Parent drug and metabolites were separated using only a Nucleosil 7.5×4.6mm 5μC<sub>18</sub>-guard cartridge as column (Alltech Assoc., Deerfield, IL), with a flow rate of 0.35mL/min, eluted with a binary solvent system consisting of 5mM NH<sub>4</sub>OAc (solvent A) and 0.5% formic acid in MeOH (solvent B). The solvent was set at 50% B for 2 minutes and increased linearly to 100% B over the following 3.5 min. From 5.5 to 6.5min, the solvent composition was lowered back down to 40% B where it was maintained for one minute (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><p id="P26" class="p p-last">Data analyses were carried out on Windows XP-based Micromass MassLynxNT<sup>®</sup>, v 4.1, software.</p></div><div id="S14" class="sec"><h3>Chemical Inhibition Experiments</h3><p id="P27" class="p p-first">Incubations were carried out using a pool of HLMs consisting of equal quantities of total microsomal protein obtained from eight different liver samples, prepared as described previously (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638485">41</a>). All incubation mixtures contained 250μg/mL microsomal protein from the HLM pool, 1 mM NADPH, 1% v/v of a 100x concentrated methanolic inhibitor stock and S-warfarin, added as substrate (at 1, 4 or 16 μM final concentration), made up to 200 μL total volume with 100 mM potassium phosphate buffer, pH 7.4. The concentrations of inhibitor used in the K<sub>I</sub> studies varied with the compound (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). All incubations were carried out in triplicate.</p><p id="P28" class="p p-last">HLMs were preincubated with (S)-warfarin, inhibitor and buffer at 37°C and 70 rpm in a water bath for 2.5 min prior to initiation of the reaction with the addition of NADPH. After 30 min incubation the reactions were quenched by the addition of 5 μL of 70% perchloric acid and 3 ng of 7-ethoxycoumarin was added as internal standard. The reaction mixtures were vortexed for 30 sec and centrifuged at 13,400 rpm for 5 min. Supernatants were then analyzed for 7-hydroxywarfarin content by HPLC using fluorescence detection. A time course study revealed that, under the experimental conditions, 7-hydroxywarfarin production was essentially linear out to 30 minutes.</p></div><div id="S15" class="sec"><h3>HPLC-Fluorescence Assay for Microsomal CYP2C9 Activity</h3><p id="P29" class="p p-first-last">HPLC was performed on a Shimadzu system equipped with two LC10ADvp pumps, a CBM-20A communication bus module, an RF-10A<sub>XL</sub> fluorescence detector and an SIL-20A<sub>HT</sub> autosampler (Shimadzu Scientific Instruments, Inc., Columbia, MD) using a Nucleosil 5μ, 4.6 × 100 mm C<sub>18</sub> HPLC column (Macherey-Nagel, Bethlehem, PA), with a flow rate of 1.3mL/min. The analytical method was isocratic, set at 38% solvent B (MeCN, solvent A = 0.5% phosphoric acid) with a run time of 10 min. The fluorescent excitation wavelength was set at 320 nm and the emission was monitored at 415 nm. Data acquisition and analyses were performed on LCsolutions<sup>®</sup> software (v 1.2, Shimadzu).</p></div><div id="S16" class="sec sec-last"><h3>Determination of Fraction Unbound (f<sub>u</sub>) in HLM and in Plasma</h3><p id="P30" class="p p-first">Protein binding of AMIO, and its metabolites, was measured in both human plasma and in pooled HLM by ultracentrifugation (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638500">42</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_510638509">43</a>) using a TLA-100 benchtop ultracentrifuge with a TLA-100 rotor (Beckman-Coulter, Palo Alto, CA). AMIO, or one of its metabolites, was spiked (as 100x concentrated methanol stocks), into samples of either HLM (0.25 mg/mL microsomal protein in 100mM KPi buffer, pH 7.4) or blank human plasma. Aliquots of 200 μL were taken from the samples and placed into polycarbonate ultracentrifugation tubes (Beckman-Coulter, no. 343775) which were either centrifuged at 100,000 rpm at 37°C for 2 hours or incubated, without centrifugation, at the same temperature for the same time span. After ultracentrifugation, a 50 μL aliquot was removed from the clear, top layer of the plasma, or the HLM supernatant, and added to the same volume of MeCN. Likewise, after remixing, 50 μL aliquots were taken from the samples which had not been ultracentrifuged and added to MeCN. A standard solution (2.5 μL), containing a mix of d<sub>2</sub>-AMIO, d<sub>4</sub>-MDEA, d<sub>4</sub>-DDEA, d<sub>2</sub>-ODAA and d<sub>2</sub>-DAA (80 ng, 80 ng, 2.5 ng, 2.5 ng and 2.5 ng, respectively), in methanol was added to each sample as internal standards. The samples were vortexed then centrifuged (13,400 rpm, 10min), and the supernatants were transferred to vials for LCMS analysis. All reactions were carried out in triplicate. Standard curves were determined for both centrifuged and uncentrifuged samples over a range from 0.010 to 20 μM. The HLM and plasma free drug fractions were calculated as the ratio of the slope of the free concentration curve (determined from the ultracentrifuged samples) to the slope of the total concentration curve (uncentrifuged samples).</p><p id="P31" class="p p-last">Protocols for the synthesis of deuterium-labeled standards and for the quantitation of AMIO and its metabolites in patient plasma samples are included in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#SD1" rid="SD1" class=" supplementary-material">Supplemental Materials</a>.</p></div></div><div id="S17" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-17"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-18">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Materials</h4><div class="sup-box half_rhythm" id="d37e886"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/bin/NIHMS523765-supplement-Supplementary_Materials.pdf">Click here to view.</a><sup>(125K, pdf)</sup></div></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-19"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-20">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank Dr. Nina Isoherranen (Univ. Washington, Seattle), for helpful discussions and Sandy Harris Research Nurse Coordinator at the University of Washington School of Medicine/Division of Cardiology, for her help in providing us with blood samples from patients undergoing warfarin/amiodarone therapy. This work was supported by the National Institutes of Health [Project Program Grant P01GM32165] to AER.</p></div></div><div id="__fn-groupid512426" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-21"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-22">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__fn-groupid512426title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first-last"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#SD1" rid="SD1" class=" supplementary-material">Supplementary Material</a> is linked to the online version of the paper at <a href="http://www.nature.com/cpt" ref="reftype=extlink&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.nature.com/cpt</a></p><p></p><p class="fn sec" id="FN3"> </p><p id="__p3" class="p p-first"><strong>Conflict of Interest</strong></p><p id="__p4" class="p p-last">The authors declared no conflict of interest.</p><p></p></div></div><div id="__ref-listid512467" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-23"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-24">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__ref-listid512467title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Vastesaeger E. Fibrin atherosclerosis. <span><span class="ref-journal">Acta Cardiol. </span>1967;<span class="ref-vol">22</span>:581–6.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/4869484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Julian DG, Camm AJ, Frangin G, Janse MJ, Munoz A, Schwartz PJ, et al.  Randomised trial of effect of amiodarone on mortality in patients with left-ventricular dysfunction after recent myocardial infarction: EMIAT. European Myocardial Infarct Amiodarone Trial Investigators. <span><span class="ref-journal">Lancet. </span>1997;<span class="ref-vol">349</span>:667–74.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/9078197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Mason JW. Amiodarone. <span><span class="ref-journal">N Engl J Med. </span>1987;<span class="ref-vol">316</span>:455–66.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/3543680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Heger JJ, Prystowsky EN, Zipes DP. Relationships between amiodarone dosage, drug concentrations, and adverse side effects. <span><span class="ref-journal">Am Heart J. </span>1983;<span class="ref-vol">106</span>:931–5.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/6613839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Marchlinski FE, Gansler TS, Waxman HL, Josephson ME. Amiodarone pulmonary toxicity. <span><span class="ref-journal">Ann Intern Med. </span>1982;<span class="ref-vol">97</span>:839–45.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/7149492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Rigas B, Rosenfeld LE, Barwick KW, Enriquez R, Helzberg J, Batsford WP, et al.  Amiodarone hepatotoxicity. A clinicopathologic study of five patients. <span><span class="ref-journal">Ann Intern Med. </span>1986;<span class="ref-vol">104</span>:348–51.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/3946978" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Dickstein G, Amikam S, Riss E, Barzilai D. Thyrotoxicosis induced by amiodarone, a new efficient antiarrhythmic drug with high iodine content. <span><span class="ref-journal">Am J Med Sci. </span>1984;<span class="ref-vol">288</span>:14–7.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/6465185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Trujillo TC, Nolan PE. Antiarrhythmic agents - Drug interactions of clinical significance. <span><span class="ref-journal">Drug Safety. </span>2000;<span class="ref-vol">23</span>:509–32.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11144659" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Funck-Brentano C, Becquemont L, Kroemer HK, Buhl K, Knebel NG, Eichelbaum M, et al.  Variable disposition kinetics and electrocardiographic effects of flecainide during repeated dosing in humans: contribution of genetic factors, dose-dependent clearance, and interaction with amiodarone. <span><span class="ref-journal">Clin Pharmacol Ther. </span>1994;<span class="ref-vol">55</span>:256–69.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/8143391" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Chitwood KK, Abdul-Haqq AJ, Heim-Duthoy KL. Cyclosporine-amiodarone interaction. <span><span class="ref-journal">Ann Pharmacother. </span>1993;<span class="ref-vol">27</span>:569–71.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/8347904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Nicolau DP, Uber WE, Crumbley AJ, 3rd, Strange C. Amiodarone-cyclosporine interaction in a heart transplant patient. <span><span class="ref-journal">J Heart Lung Transplant. </span>1992;<span class="ref-vol">11</span>:564–8.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/1610865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Funck-Brentano C, Jacqz-Aigrain E, Leenhardt A, Roux A, Poirier JM, Jaillon P. Influence of amiodarone on genetically determined drug metabolism in humans. <span><span class="ref-journal">Clin Pharmacol Ther. </span>1991;<span class="ref-vol">50</span>:259–66.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/1914360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Thi L, Shaw D, Bird J. Warfarin potentiation: a review of the “FAB-4” significant drug interactions. <span><span class="ref-journal">Consult Pharm. </span>2009;<span class="ref-vol">24</span>:227–30.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19555137" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Lip GY, Lim HS. Atrial fibrillation and stroke prevention. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:981–93.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17945152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Kerin NZ, Blevins RD, Goldman L, Faitel K, Rubenfire M. The incidence, magnitude, and time course of the amiodarone-warfarin interaction. <span><span class="ref-journal">Arch Intern Med. </span>1988;<span class="ref-vol">148</span>:1779–81.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/3401099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Sanoski CA, Bauman JL. Clinical observations with the amiodarone/warfarin interaction: dosing relationships with long-term therapy. <span><span class="ref-journal">Chest. </span>2002;<span class="ref-vol">121</span>:19–23.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11796427" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">O’Reilly RA, Trager WF, Rettie AE, Goulart DA. Interaction of amiodarone with racemic warfarin and its separated enantiomorphs in humans. <span><span class="ref-journal">Clin Pharmacol Ther. </span>1987;<span class="ref-vol">42</span>:290–4.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/3621782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Heimark LD, Wienkers L, Kunze K, Gibaldi M, Eddy AC, Trager WF, et al.  The mechanism of the interaction between amiodarone and warfarin in humans. <span><span class="ref-journal">Clin Pharmacol Ther. </span>1992;<span class="ref-vol">51</span>:398–407.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/1563209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Kumar V, Locuson CW, Sham YY, Tracy TS. Amiodarone analog-dependent effects on CYP2C9-mediated metabolism and kinetic profiles. <span><span class="ref-journal">Drug Metab Dispos. </span>2006;<span class="ref-vol">34</span>:1688–96.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16815961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Ohyama K, Nakajima M, Suzuki M, Shimada N, Yamazaki H, Yokoi T. Inhibitory effects of amiodarone and its N-deethylated metabolite on human cytochrome P450 activities: prediction of in vivo drug interactions. <span><span class="ref-journal">Br J Clin Pharmacol. </span>2000;<span class="ref-vol">49</span>:244–53.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2014912/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/10718780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Rettie AE, Korzekwa KR, Kunze KL, Lawrence RF, Eddy AC, Aoyama T, et al.  Hydroxylation of warfarin by human cDNA-expressed cytochrome P-450: a role for P-4502C9 in the etiology of (S)-warfarin-drug interactions. <span><span class="ref-journal">Chem Res Toxicol. </span>1992;<span class="ref-vol">5</span>:54–9.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/1581537" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Marchiset D, Bruno R, Djiane P, Cano JP, Benichou M, Serradimigni A. Amiodarone and desethylamiodarone elimination kinetics following withdrawal of long-term amiodarone maintenance therapy. <span><span class="ref-journal">Biopharm Drug Dispos. </span>1985;<span class="ref-vol">6</span>:209–15.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/4005397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Naganuma M, Shiga T, Nishikata K, Tsuchiya T, Kasanuki H, Fujii E. Role of desethylamiodarone in the anticoagulant effect of concurrent amiodarone and warfarin therapy. <span><span class="ref-journal">J Cardiovasc Pharmacol Ther. </span>2001;<span class="ref-vol">6</span>:363–7.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11907638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Ha HR, Bigler L, Wendt B, Maggiorini M, Follath F. Identification and quantitation of novel metabolites of amiodarone in plasma of treated patients. <span><span class="ref-journal">Eur J Pharm Sci. </span>2005;<span class="ref-vol">24</span>:271–9.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/15734293" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Harris L, Roncucci RR.  <span class="ref-journal">Amiodarone: pharmacology, pharmacokinetics, toxicology, clinical effects.</span> Médecine et sciences internationales; Paris: 1986. </span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Locuson CW, 2nd, Wahlstrom JL, Rock DA, Jones JP. A new class of CYP2C9 inhibitors: probing 2C9 specificity with high-affinity benzbromarone derivatives. <span><span class="ref-journal">Drug Metab Dispos. </span>2003;<span class="ref-vol">31</span>:967–71.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12814975" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lucas AN, Tanol M, McIntosh MP, Rajewski RA. Preparation and purification of desethylamiodarone hydrochloride. <span><span class="ref-journal">Synthetic Communications. </span>2006;<span class="ref-vol">36</span>:3371–6.</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Waldhauser KM, Torok M, Ha HR, Thomet U, Konrad D, Brecht K, et al.  Hepatocellular toxicity and pharmacological effect of amiodarone and amiodarone derivatives. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>2006;<span class="ref-vol">319</span>:1413–23.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16971508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Snead AN, Miyakawa M, Tan ES, Scanlan TS. Trace amine-associated receptor 1 (TAAR1) is activated by amiodarone metabolites. <span><span class="ref-journal">Bioorg Med Chem Lett. </span>2008;<span class="ref-vol">18</span>:5920–2.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3609712/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/18752950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Wendt B, Ha HR, Hesse M. Synthesis of two metabolites of the antiarrythmicum amiodarone. <span><span class="ref-journal">Helvetica Chimica Acta. </span>2002;<span class="ref-vol">85</span>:2990–3001.</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Deng P, You TG, Chen XY, Yuan T, Huang HH, Zhong DF. Identification of Amiodarone Metabolites in Human Bile by Ultra Performance Liquid Chromatography/Quadrupole Time-of-Flight Mass Spectrometry. <span><span class="ref-journal">Drug Metab Dispos. </span>2011</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21398391" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Lalloz MR, Byfield PG, Greenwood RM, Himsworth RL. Binding of amiodarone by serum proteins and the effects of drugs, hormones and other interacting ligands. <span><span class="ref-journal">J Pharm Pharmacol. </span>1984;<span class="ref-vol">36</span>:366–72.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/6146666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">U.S. Food and Drug Administration. <span class="ref-journal">Draft Guidance.</span> Rockville, MD: 2006. Guidance for Industry Drug Interaction Studies - Study Design, Data Analysis, and Implications for Dosing and Labeling.</span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Yeung CK, Fujioka Y, Hachad H, Levy RH, Isoherranen N. Are circulating metabolites important in drug-drug interactions?: Quantitative analysis of risk prediction and inhibitory potency. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2011;<span class="ref-vol">89</span>:105–13.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3474849/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21124313" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Blanchard N, Richert L, Coassolo P, Lave T. Qualitative and quantitative assessment of drug-drug interaction potential in man, based on Ki, IC50 and inhibitor concentration. <span><span class="ref-journal">Curr Drug Metab. </span>2004;<span class="ref-vol">5</span>:147–56.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/15078192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Ito K, Chiba K, Horikawa M, Ishigami M, Mizuno N, Aoki J, et al.  Which concentration of the inhibitor should be used to predict in vivo drug interactions from in vitro data? <span><span class="ref-journal">AAPS PharmSci. </span>2002;<span class="ref-vol">4</span>:E25.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2751314/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12645997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Obach RS. Nonspecific binding to microsomes: impact on scale-up of in vitro intrinsic clearance to hepatic clearance as assessed through examination of warfarin, imipramine, and propranolol. <span><span class="ref-journal">Drug Metab Dispos. </span>1997;<span class="ref-vol">25</span>:1359–69.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/9394025" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Hanson KL, VandenBrink BM, Babu KN, Allen KE, Nelson WL, Kunze KL. Sequential metabolism of secondary alkyl amines to metabolic-intermediate complexes: opposing roles for the secondary hydroxylamine and primary amine metabolites of desipramine, (s)-fluoxetine, and N-desmethyldiltiazem. <span><span class="ref-journal">Drug Metab Dispos. </span>2010;<span class="ref-vol">38</span>:963–72.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879960/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/20200233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Mori K, Hashimoto H, Takatsu H, Tsuda-Tsukimoto M, Kume T. Cocktail-substrate assay system for mechanism-based inhibition of CYP2C9, CYP2D6, and CYP3A using human liver microsomes at an early stage of drug development. <span><span class="ref-journal">Xenobiotica. </span>2009;<span class="ref-vol">39</span>:415–22.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19480547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">European Medicines Agency. <span class="ref-journal">Guideline on the Investigation of Drug Interactions.</span> Canary Wharf; London, U.K: 2010. </span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Sadeque AJ, Eddy AC, Meier GP, Rettie AE. Stereoselective sulfoxidation by human flavin-containing monooxygenase. Evidence for catalytic diversity between hepatic, renal, and fetal forms. <span><span class="ref-journal">Drug Metab Dispos. </span>1992;<span class="ref-vol">20</span>:832–9.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/1362935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Nakai D, Kumamoto K, Sakikawa C, Kosaka T, Tokui T. Evaluation of the protein binding ratio of drugs by a micro-scale ultracentrifugation method. <span><span class="ref-journal">J Pharm Sci. </span>2004;<span class="ref-vol">93</span>:847–54.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/14999723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Templeton IE, Thummel KE, Kharasch ED, Kunze KL, Hoffer C, Nelson WL, et al.  Contribution of itraconazole metabolites to inhibition of CYP3A4 in vivo. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">83</span>:77–85.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488349/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17495874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3900249&amp;issue-id=232743&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#__abstractid206947title">Abstract</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#S1title">Introduction</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#S2title">Results</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#S9title">Discussion</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#S10title">Methods</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#S17title">Supplementary Material</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#S18title">Acknowledgments</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#__fn-groupid512426title">Footnotes</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#__ref-listid512467title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="try-button"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/?report=reader"><img src="./PMC3900249_files/3838809" alt="PubReader format: click here to try"></a></div><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/epub/">ePub (beta)</a></li> | <li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/pdf/nihms523765.pdf">PDF (948K)</a></li> | <li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" data-citationid="PMC3900249" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3900249%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./PMC3900249_files/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3900249%2F&amp;text=Warfarin-Amiodarone%20Drug-Drug%20Interactions%3A%20Determination%20of%20%5BI%5Du%2FKI%2Cu%20for%20Amiodarone%20and%20its%20Plasma%20Metabolites" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./PMC3900249_files/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3900249%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./PMC3900249_files/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22398967" data-aiid="3900249" data-aid="3900249" data-iid="232743" data-domainid="319" data-domain="nihpa" data-accid="PMC3900249" data-md5="556af9bb50330cdaf808dacfe514774e"><img src="./PMC3900249_files/3631476" align="absmiddle" alt="Loading..."> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="./PMC3900249_files/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="http://www.ncbi.nlm.nih.gov/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/pmc/">PubMed Central (PMC)</a></span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="http://www.ncbi.nlm.nih.gov/sites/ehelp?&amp;Ncbi_App=pmc&amp;Db=pmc&amp;Page=literature&amp;Snapshot=/projects/PMC/PMCViewer@4.35&amp;Time=2016-06-23T14:26:26-04:00&amp;Host=ptpmc201">Write to the Help Desk</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="./PMC3900249_files/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Copyright</a> | <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="http://www.ncbi.nlm.nih.gov/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="http://www.ncbi.nlm.nih.gov/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="./PMC3900249_files/InstrumentPageStarterJS.js"> </script>    
    <script type="text/javascript" src="./PMC3900249_files/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 63.5938px; left: 408.188px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: none;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900249/#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: none;">Turn off</a></div></div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=nihpa&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC3900249/&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- F4FC7E8376854741_0014SID /projects/PMC/PMCViewer@4.35 ptpmc201 v4.1.r480124 Mon, Sep 28 2015 02:46:19 -->

<script type="text/javascript" src="./PMC3900249_files/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1" style="display: none;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><ul class="print-log"><li></li></ul><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-family: STIXSizeOneSym, sans-serif;"></div></div></body></html>